cdk/assets/scheduled_task_exec/index.js

```diff
@@ -1,38 +1,41 @@
-const aws = require('aws-sdk')
+const aws = require('aws-sdk');
```

Comment:
I'm not sure if this project is built using es8, but there are some nifty new features if you're interested in switching over. That's probably not something you have time for right now, but it includes new syntax for importing and exporting + better module management.

Ex: `import was from 'aws-sdk'`

Reply:
Thanks for the suggestion. not sure if it was specifically es8 but IIRC I had to hold off on updating absolutely everything because of something related to mixing js and ts in the project.